src/applications/vaos/services/appointment/index.js

```diff
@@ -115,8 +115,20 @@ function hasPartialResults(response) {
  * @param {String} endDate Date in YYYY-MM-DD format
  * @returns {Appointment[]} A FHIR searchset of booked Appointment resources
  */
-export async function getBookedAppointments({ startDate, endDate }) {
+export async function getBookedAppointments({
+  startDate,
+  endDate,
+  useV2 = false,
+}) {
   try {
+    if (useV2) {
+      const appointments = await getAppointments(startDate, endDate, [
+        'booked',
```

Comment:
I'm going to have to ask about how we deal with cancelled appointments/requests. We probably need to include `cancelled` in here to pull those down, too. But then we'd also have to filter them so we didn't show cancelled requests in the wrong place.

I'll make a ticket for handling cancelled appointments/requests, no need to make any changes right now.